packages/react-components/src/Concepts/Accessibility/AccessibleComponents.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/addon-docs';
+import LinkTo from '@storybook/addon-links/react';
+import { ScenariosListLink } from '../../AccessibilityScenarios/utils';
+
+<Meta title="Concepts/Developer/Accessibility/Components" />
+
+## Components accessibility
+
+Fluent UI components are designed to support various aspects of accessibility by default, so that they can be used with different input methods (mouse, touch, keyboard, screen readers) as well as fulfill different rendering and layout requirements (theming, zoom, contrast).
+
+The [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/) and [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.2/) are community-driven standards for accessible web pages and applications. They provide a basic framework allowing consistent usage of web pages and applications with or without assistive technologies.
+
+Fluent UI components aim to fully respect both WCAG and ARIA and the Fluent UI team is working with the community to improve them.
+
+Using components themselves does not guarantee the application or a page to be accessible. Please see [Accessible Experiences](?path=/docs/concepts-developer-accessibility-experiences--page) page for further points on achieving good level of usability.
+
+## Scope
+
+Fluent UI components fulfill following requirements:
+
+- **DOM structure** that provides **semantical value** either by using correct element types or roles
+- **ARIA attributes** that are valid for elements/roles used and provide correct state information about the component or element
+- **Keyboard navigation** (navigation by tabbing, arrow keys, pagination or letter keys, click and right click (Enter/Space, Shift+F10), and close (Escape)) applied based on the component
+- **Screen reader navigation** (Virtual Cursor/Browse mode/Scan mode/VoiceOver keys)
+- **Touch interaction**
```

Comment:
I am not aware of any clear requirement of this one, or whether we even kind of know we support it, let's leave it out for now

```suggestion
```

Reply:
it is a requirement - maybe we need to improve our test coverage there - but components need to work with touch